### ISSUE TITLE
Icons at same location

### ISSUE THREAD

**Issue by pmoitech** [googlearchive/js-marker-clusterer#35]
Hello, thanks for the demos for markerCluterer.

I am facing an issue when I have 2 or more icons at the same location (same coordinates). In this case the markerer show only the icon with the number and it is not possible to see the icons inside, is there a solution for this??

Thanks

**Comment by ramonrovirosa** [comment]
Having the same issue, @pmoitech, did you find a solution to this issue?

I found this on stackoverflow: http://stackoverflow.com/questions/20490654/more-than-one-marker-on-same-place-markerclusterer

**Comment by kshipp** [comment]
+1

This needs to be fixed.
I was able to adjust the LatLon enough to make the markers place in a different location, enough to keep them from disappearing on the markerCluster zoom.

**Comment by ldrrp** [comment]
MaxZoom ?

**Comment by christian-felix** [comment]
are there some progress on this issue?

**Comment by ldrrp** [comment]
nope, looks like google abandoned this project. its been stale for years.

**Comment by christian-felix** [comment]
I had solved this issue by using an div container in conjunction with an simple paginator. 
![marker](https://user-images.githubusercontent.com/5967862/52475975-7ee04e80-2b9d-11e9-93fc-65d8e110cc8b.png)

The idea with the spiderfier were also an solution but while more than 20 entries on the same location not really clearly.
